src/core/ddsi/src/ddsi_discovery_spdp.c

```diff
@@ -182,7 +181,7 @@ void ddsi_get_participant_builtin_topic_data (const struct ddsi_participant *pp,
     if (ddsrt_gethostname(node, sizeof(node)-1) < 0)
 #endif
       (void) ddsrt_strlcpy (node, "unknown", sizeof (node));
-    size = strlen(node) + strlen(DDS_VERSION) + strlen(DDS_HOST_NAME) + strlen(DDS_TARGET_NAME) + 4; /* + ///'\0' */
+    size = strlen(node) + strlen(DDS_VERSION) + strlen(DDS_HOST_NAME) + strlen(DDS_TARGET_NAME) + 4; // + '\0';
```

Comment:
That should now read `// 3x'/' + '\0'` or something similar

Reply:
Ah, now it makes sense. I wasn't exactly sure how to grok the original comment. Resolved in 92fe0c90a8b2571f8ad95aa7b4d2332dc4b8daa7.